Our sysvinit spawns an extra agetty for each cmdline console=, so let's
not bother adding non serial ones. Don't at pointless to us TERM= either.

Clean up boot messages, by removing rather uninteresting console= note, too.

--- palo-2.24/ipl/ipl.c.vanilla	2023-12-16 22:42:27.091651840 +0100
+++ palo-2.24/ipl/ipl.c	2023-12-16 22:45:31.157643983 +0100
@@ -629,11 +629,12 @@
     /* add the right console= if there isn't one yet */
     if (strstr(commandline, " console=") == 0)
     {
+	/*
 	printf("\nInformation: No console specified on kernel command line."
 		" This is normal.\nPALO will choose the console currently"
 		" used by firmware ");
+	*/
 
-        chk_strcat(commandline, " console=", CMDLINELEN, &ok);
 	if (pdc_cons_duplex())
 	{
             int is_mux;
@@ -643,18 +644,19 @@
             if(pdc_cons_mux(&is_mux) != PDC_OK)
                 printf("Information: The PDC calls to query the console device failed. Assuming console=ttyS0\n");
 
+	    chk_strcat(commandline, " console=", CMDLINELEN, &ok);
             if(is_mux)
 	        chk_strcat(commandline, "ttyB0", CMDLINELEN, &ok);
             else
 	        chk_strcat(commandline, get_default_serial_console(), CMDLINELEN, &ok);
 
-	    if (strstr(commandline, " TERM=") == 0)
-	        chk_strcat(commandline, " TERM=vt102", CMDLINELEN, &ok);
+	    //if (strstr(commandline, " TERM=") == 0)
+	        //chk_strcat(commandline, " TERM=vt102", CMDLINELEN, &ok);
 	}
 	else
 	{
 	    printf("(graphics).\n");
-	    chk_strcat(commandline, "tty0", CMDLINELEN, &ok);
+	    //chk_strcat(commandline, "tty0", CMDLINELEN, &ok);
 	    if (strstr(commandline, " sti=") == 0)
 	    {
 		struct {
@@ -685,8 +687,8 @@
 	    }
 	    if (strstr(commandline, " sti_font=") == 0)
 	        chk_strcat(commandline, " sti_font=VGA8x16", CMDLINELEN, &ok);
-	    if (strstr(commandline, " TERM=") == 0)
-	        chk_strcat(commandline, " TERM=linux", CMDLINELEN, &ok);
+	    //if (strstr(commandline, " TERM=") == 0)
+	        //chk_strcat(commandline, " TERM=linux", CMDLINELEN, &ok);
 	}
     }
 
